README.md

```diff
@@ -1,19 +1,30 @@
 # The Front-End Playbook
 
-The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this playbook are to be interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
+This repo contains The Front-End Playbook. It details how we run software development and how we make web and mobile products together. It's filled with things we've learned based on our own experience and study of others' experiences.
 
-## What's this?
+## Table of contents
 
-This repo contains The Front-End Playbook. It details how we run software development and how we make web and mobile products together. It's filled with things we've learned based on our own experience and study of others' experiences.
+* [Introduction](#whats-this)
+* [Accessibility](accessibility/README.md)
+* [CSS](css/README.md)
+* [Git](git/README.md)
+* [JavaScript](javascript/README.md)
+* [Markup](markup/README.md)
+* [Performance](performance/README.md)
+* [Practices](practices/README.md)
+* [Security](security/README.md)
+* [Writing](writing/README.md)
 
-See "[Changing the laws of engineering with pull requests](https://www.youtube.com/watch?v=YIpNpptGX6Q)" for an in depth explanation of how developing a playbook like this is of benefit.
+## What's this?
 
 This is a living document that we contribute to in a _public_ GitHub repo. Reasons for doing this in the open include (but are not limited to):
 
 1. Interacting with and learning from others. Receiving contributions from people who don't work here can help us, providing learning opportunities that we would not receive otherwise - for example, see [this contribution](https://github.com/springernature/frontend-playbook/pull/48#issuecomment-236139605) from @rowanmanning.
 
 1. Providing a showcase for our work/ethics. This is _really_ useful when hiring people (for both parties). We've had very positive feedback from interviewees - it's a great recruiting tool. It also means that people are quickly up and running when they join.
 
+See "[Changing the laws of engineering with pull requests](https://www.youtube.com/watch?v=YIpNpptGX6Q)" for an in depth explanation of how developing a playbook like this is of benefit.
+
 ### Where do I start?
```

Comment:
I wonder if this should be removed now?
I presume that you 'start' at this README, and then navigate off via the TOC.

Reply:
Why just this line? Confused.

Reply:
No not the line, I meant the `Where do I start` section.

Reply:
oh! that makes more sense. 😂